packages/performance-tests/package.json

```diff
@@ -0,0 +1,60 @@
+{
+  "name": "transformer-performance-tests",
+  "private": true,
+  "license": "MIT",
+  "version": "0.1.0",
+  "scripts": {
+    "build": "tsc 1>&2",
+    "build:ci": "npm run -s build",
+    "clean": "rimraf lib",
+    "lint": "eslint --max-warnings 0 \"./src/**/*.ts\" 1>&2",
+    "test": "mocha --delay --timeout 300000 --require ts-node/register test/**/*.test.ts",
+    "test-reg": "mocha --delay --timeout 300000 --require ts-node/register test/**/TransformerRegression.test.ts",
```

Comment:
can you update this and its references to `test-regression`? `reg` is kind of hard to understand without reading the rest of the line